build/wrapper.cc

```diff
@@ -482,7 +482,7 @@ JLCXX_MODULE define_julia_module(jlcxx::Module& mod)
         .method("Hex", &TaskID::Hex);
 
     mod.method("initialize_driver", &initialize_driver);
-    mod.method("shutdown_driver", &shutdown_driver);
+    mod.method("_shutdown_driver", &shutdown_driver);
```

Comment:
why the underscore? I don't see this being called anywhere?

Reply:
See this commit: https://github.com/beacon-biosignals/Ray.jl/pull/158/commits/4978bf1801b761d7df7987a4c7bf2e28140eb80e

This is being called from `ray_julia_jll.shutdown_driver()`